systests/microprofile/client/async/pom.xml

```diff
@@ -76,6 +76,12 @@
                 <version>${cxf.johnzon.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
think you can use JsonbJaxrsProvider from johnzon-jsonb instead of importing this one which does not use jsonb, would be more consistent with MP probably?

Reply:
Sure, done, thanks!